app/schemas.py

```diff
@@ -267,10 +267,13 @@ def handle_personalisation_property(self, in_data):
     def handle_template_merge(self, in_data):
         if in_data.get('personalisation'):
             from notifications_utils.template import Template
-            merged = Template(in_data['template'], in_data['personalisation']).replaced
-            in_data['body'] = merged
+            merged = Template(in_data['template'], in_data['personalisation'])
+            in_data['body'] = merged.replaced
```

Comment:
i use it on the next line - so need to save it rather than the previous one-liner. happy to rename it to `template` tho